src/Application/Lockers/Commands/AddLocker/AddLockerCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Application.Common.Interfaces;
+using Application.Common.Models.Dtos.Physical;
+using AutoMapper;
+using Domain.Entities.Physical;
+using Domain.Exceptions;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Lockers.Commands.AddLocker;
+
+public record AddLockerCommand : IRequest<LockerDto>
+{
+    public string Name { get; init; }
+    public string Description { get; init; }
+    public string RoomId { get; init; }
```

Comment:
Id is a Guid, not a string

Reply:
should be*

Reply:
no no, im doing this to be able to check the validity, check the validator

Reply:
When you declare the RoomId as Guid, the framework validate if automatically for you and return 400 if what the user types in is not a Guid